src/store/store.js

```diff
@@ -5,7 +5,7 @@ export const store = createStore({
     name: "Vue",
     workout: {
       duration: {
-        active: "30",
+        active: "60",
```

Comment:
This is only the default value and I'd prefer it to stay at 30s - feel free to open an issue asking about a "default duration defined within each workout" so we can lift it to 60s for your ones :)

Reply:
I appreciate that the default value is preferred to stay at 30s. I will open an issue to request an increase to 60s for my specific workouts.